src/main/java/org/kainos/ea/cli/DeliveryEmployee.java

```diff
@@ -0,0 +1,18 @@
+package org.kainos.ea.cli;
+
+public class DeliveryEmployee extends Employee{
+
+    public DeliveryEmployee(int employeeID, double salary, String bankAccountNumber, String NINumber, int deliveryEmployeeID) {
+        super(employeeID, salary, bankAccountNumber, NINumber);
+        setDeliveryEmployeeID(deliveryEmployeeID);
+    }
+    private int deliveryEmployeeID;
+
+    public int getDeliveryEmployeeID() {
+        return deliveryEmployeeID;
+    }
```

Comment:
do you need a delivery employee ID? if you use this in the db it will prevent you from using the key integrity to only allow an EmployeeID to exist in the DeliveryEmployee table multiple times